src/Makefile.OCaml

```diff
@@ -72,7 +76,12 @@ endif
 
 .PHONY: macuimaybe
 ifeq ($(OSARCH), Darwin)
-  macuimaybe: macui
+  ifeq ($(XCODEBUILD), 0)
+    macuimaybe: macui
+  else
+    macuimaybe:
+      $(info Not the xcode version of xcodebuild, macOS native GUI can not be build.)
```

Comment:
I would suggest
```
Not building macOS native GUI because XCode is not installed.
```
if you think that's accurate of course.

Reply:
Yes it is, a lot nicer that mine